tools/coreml/converter/_layers.py

```diff
@@ -220,6 +220,39 @@ def convert_activation(net, node, module, builder):
                            output_name = output_name)
 
 
+def convert_leakyrelu(net, node, module, builder):
+    """Convert an leakyrelu layer from mxnet to coreml.
+
+    Parameters
+    ----------
+    network: net
+        A mxnet network object.
+
+    layer: node
+        Node to convert.
+
+    module: module
+        An module for MXNet
+
+    builder: NeuralNetworkBuilder
+        A neural network builder object.
+    """
+    input_name, output_name = _get_input_output_name(net, node)
```

Comment:
Extremely minor: A spare line after documentation and before code starts will be good. (I am aware that this practice is not being followed in other areas of the code in this file but lets make this code better?)

Reply:
Changed in a1e937d